go.mod

```diff
@@ -98,81 +98,81 @@ require (
-	k8s.io/controller-manager => k8s.io/controller-manager v0.22.3
-	k8s.io/cri-api => k8s.io/cri-api v0.22.3
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.22.3
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.22.3
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.22.3
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.22.3
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.22.3
-	k8s.io/kubectl => k8s.io/kubectl v0.22.3
-	k8s.io/kubelet => k8s.io/kubelet v0.22.3
-	k8s.io/kubernetes => k8s.io/kubernetes v1.22.2
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.22.3
-	k8s.io/metrics => k8s.io/metrics v0.22.3
-	k8s.io/mount-utils => k8s.io/mount-utils v0.22.3
-	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.22.3
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.22.3
-	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.9.2
-	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.6.1
+	k8s.io/api => k8s.io/api v0.23.0
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.23.0
+	k8s.io/apimachinery => k8s.io/apimachinery v0.23.0
+	k8s.io/apiserver => k8s.io/apiserver v0.23.0
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.23.0
+	k8s.io/client-go => k8s.io/client-go v0.23.0
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.23.0
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.23.0
+	k8s.io/code-generator => k8s.io/code-generator v0.23.0
+	k8s.io/component-base => k8s.io/component-base v0.23.0
+	k8s.io/component-helpers => k8s.io/component-helpers v0.23.0
+	k8s.io/controller-manager => k8s.io/controller-manager v0.23.0
+	k8s.io/cri-api => k8s.io/cri-api v0.23.0
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.23.0
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.23.0
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.23.0
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.23.0
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.23.0
+	k8s.io/kubectl => k8s.io/kubectl v0.23.0
+	k8s.io/kubelet => k8s.io/kubelet v0.23.0
+	k8s.io/kubernetes => k8s.io/kubernetes v1.23.0
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.23.0
+	k8s.io/metrics => k8s.io/metrics v0.23.0
+	k8s.io/mount-utils => k8s.io/mount-utils v0.23.0
+	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.23.0
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.23.0
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.11.0
+	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.6.2
 )
 
 // Other pinned deps
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible
 	github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.40.0
 	github.com/mtrmac/gpgme => github.com/mtrmac/gpgme v0.1.1
-	github.com/openshift-kni/debug-tools => github.com/openshift-kni/debug-tools v0.1.3-0.20211201134627-ad03be247ef2 // main
```

Comment:
this pinning was removed on purpose ?

Reply:
Yes, we can just consume the `v0.1.3` tag.

Reply:
yes, we can just consume the `v0.1.3` (or equivalent `v0.1.4`) tag.